doc/changes/changes_0.15.0.md

```diff
@@ -1,9 +1,12 @@
-# Script-Languages-Container-Tool 0.15.0, released t.b.d.
+# Script-Languages-Container-Tool 0.15.0, released 2022-08-29
 
-Code name: t.b.d.
+Code name: Initial API layer and support for stream output of tests to log files.
 
 ## Summary 
-t.b.d.
+
+This release streams the output of running tests to the logfile during test execution. Also, a new API module 
```

Comment:
I think, first the --path-in-bucket bug, then the API and last logfile. The bug fix is actually the most important thing here, because it can be really anoying